Automattic/jetpack
# Update unpurchased backup card state my jetpack

## Proposed changes:

* Replace dropdown menu with the product logo
* Make sure all cards are the same height
* Pull in product data to Backup card
* Display site data to Backup card when card is in unpurchased state
* Set up infrastructure to handle the purchase state next

### Other information:


## Jetpack product discussion

P2: p1HpG7-oSY-p2
Figma: u7alAliUsO2ePGqbrxcGwC-fi-3813%3A7585

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Make sure to test this PR in conjunction with this diff: D123259-code
2. Get your local environment running or test this branch using the Jetpack beta plugin
3. Make sure your site has some pages/posts/images/etc. What can be displayed on the card are Pages, Posts, Comments, Images, Videos, and Audio files
4. Without a Backup plan, go to `/wp-admin/admin.php?page=my-jetpack`
5. Make sure the loading state looks alright for the site data on the card

NOTE: I wanted to do more than just display "Loading...", but it looks like that's what is normal around the plugin for now so we'll probably want to update all those at once if we do end up doing so

![image](https://github.com/Automattic/jetpack/assets/65001528/83046f5b-8f81-43f4-85cb-4a9702de5f89)

6. Once the card loads your site's data, make sure it looks like the Figma comps and that it scales well to mobile (should switch to showing 2 items on certain screen widths)

![image](https://github.com/Automattic/jetpack/assets/65001528/9c035e8c-993f-4789-bb4c-181adce29285)

7. If you want to test out various values, you can manually edit the values on the diff you patched on your sandbox in the `wp-content/lib/jetpack-product-data/class.backups.php` file lines 128-144

8. Purchase VaultPress backup and go back to `/wp-admin/admin.php?page=my-jetpack`, make sure the card looks normal. We will be adding a purchased state for the data later in a separate PR

![image](https://github.com/Automattic/jetpack/assets/65001528/c5e2ade0-5cee-4e1f-8c0c-eedbef009647)
